core/common/pom.xml

```diff
@@ -123,6 +123,11 @@
       <artifactId>curator-test</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Is this supposed to be non-test dependency? Then it should be moved to a different section.

Also, @calvinjia do we want to add a hadoop dep?

Reply:
Done, move to the non-test section.